ughubHelpContents.py

```diff
@@ -52,6 +52,12 @@
 			]
 		},
 
+		 {
+            "name": "uninstall",
+            "usage": "uninstall NAME",
+            "description": "Uninstall a given package from the filesystem\n"
```

Comment:
Please adjust the usage so that it matches 'install':
"usage": "uninstall PACKAGE_1 [PACKAGE_2 [PACKAGE_3 [...]]] [OPTIONS]",
Also please adjust the description:
"Uninstalls the specified packages and removes their contents from the file system"
Please add a description for the "-f, --force" option.